examples/src/bin/temperature_sensor.rs

```diff
@@ -0,0 +1,33 @@
+//! This example uses the internal temperature sensor to measure the chip's temperature
```

Comment:
Can you move this to `qa-test` instead? There isn't much benefit to this as an example (over what the docs provide), but adding a temp sens to HIL test might be tricky... or useful to detecting a fire in the server room :D